examples/server-side-rendering/angular.json

```diff
@@ -0,0 +1,141 @@
+{
```

Comment:
This is the new `.angular-cli.json` introduced by the migration to `@angular/cli@^7`